addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -521,9 +528,10 @@ def message_new(self, msg, custom_values=None):
         defaults = {
             'name': msg.get('subject') or _("No Subject"),
             'partner_name': val,
-            'email_from': msg.get('from'),
             'partner_id': msg.get('author_id', False),
         }
+        if msg.get('from') and not self._email_is_blacklisted(msg.get('from')):
+            defaults['email_from'] = msg.get('from')
```

Comment:
I am wondering, if this is even needed? :thinking: in write method you delete the value if the email is blacklisted.

Reply:
But these default are also used for the create which doesn't use write AFAIK.